Delicate_Flowers_Vignette-AR.qmd

```diff
@@ -0,0 +1,222 @@
+# Growing in Harsh Conditions: A Study of Confederate Women during the Civil War
+
+### Addison Horton — Hist 8510 — Spring 2024
+
+Early in the semester, I came across a number of diaries written by Confederate women during the American Civil War. I was excited about the prospect of using this material but quickly realized that it was far too large a corpus for me to read in its entirety in the span of a few months. So, I did what any digital historian would do and turned to digital tools. Through word vector analysis and topic modeling, I was able to identify age as a crucial consideration when studying elite white women's reactions to the war.
+
+#### Prepping the Data
+
+The first thing I did was load the necessary packages for word vector analysis and topic modeling.
+
+```{r}
+library(tidyverse)
+library(tidytext) 
+library(readtext)
+library(widyr)
+library(SnowballC)
+library(tidytext)
+library(tidyverse)
+library(readtext)
+library(tm)
+library(topicmodels)
+library(readr)
+library(ggplot2)
+devtools::install_github("bmschmidt/wordVectors")
+library(wordVectors)
+library(tidyverse)
```

Comment:
Do you need these within your document? Probably not. They have to be loaded bu they don't have to be visible and aren't necessary for your argument. If you add #| echo: false to the top of blocks like this it will prevent it from showing up in the final rendered version. The code will still run and libraries will still load, but you won't see it in the rendered document.